extensions/firmware/org.eclipse.smarthome.firmware.filesystem.test/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
```

Comment:
The Eclipse IDE is the official IDE used by the project.
We provide a Oomph setup file (https://github.com/eclipse/smarthome/blob/master/targetplatform/EclipseSmartHome.setup) to ensure that every developer is using the same formatting rules etc.
So, we will keep all that Eclipse IDE specific files.
If "developers in ESH use IntelliJ" that developers need to ensure, that all their changes does use the correct formatting rules etc. But this is up to them.